docs/index.bs

```diff
@@ -2223,15 +2231,15 @@ spec: webappsec-referrer-policy; urlPrefix: https://w3c.github.io/webappsec-refe
       : Output
       :: none
 
-      1. Assert: |jobQueue| [=queue/is not empty=].
-      1. [=Queue a task=] to run these steps:
+      1. [=Enqueue the following steps=] to the [=service worker manager=]:
+          1. Assert: |jobQueue| [=queue/is not empty=].
           1. Let |job| be the first [=queue/item=] in |jobQueue|.
-          1. If |job|'s [=job type=] is *register*, run [=Register=] with |job| [=in parallel=].
-          1. Else if |job|'s [=job type=] is *update*, run [=Update=] with |job| [=in parallel=].
+          1. If |job|'s [=job type=] is *register*, run [=Register=] with |job|.
+          1. Else if |job|'s [=job type=] is *update*, run [=Update=] with |job|.
 
               Note: For a register job and an update job, the user agent delays queuing a task for running the job until after a {{Document/DOMContentLoaded}} event has been dispatched to the document that initiated the job.
 
-          1. Else if |job|'s [=job type=] is *unregister*, run [=Unregister=] with |job| [=in parallel=].
+          1. Else if |job|'s [=job type=] is *unregister*, run [=Unregister=] with |job|.
```

Comment:
This is a good step. Eventually I'd like to get rid of the "job" concept entirely, and replace it with appending steps to the appropriate parallel queue.

Reply:
That'd mean the implementations should create and maintain a separate thread for each scope. That may be an ideal design which allows parallel register/update/unregister even, but a single (off-the-main) thread executing the jobs from all the job queues is what we currently have. I also would like to hear what implementers think on this.